extensions/amp-story-interactive/amp-story-interactive.md

```diff
@@ -167,13 +165,7 @@ POST_BODY = {'option_selected': 1}
 Response: No response necessary
 [/sourcecode]
 
-Backends need to be specified on the necessary components (binary-poll, poll, quiz), and can be deployed by publishers, tools or others. Available free-to-use backends are:
-
-- Google hosted: _coming soon_
-
-[tip type="note"]
-Before setting up a backend, consider if the already existing backends satisfy your requirements.
-[/tip]
+Backends need to be specified on the necessary components (binary-poll, poll, quiz), and can be deployed by publishers, tools or others.
```

Comment:
In a separate PR, can we work on providing some very simple documentation on what the JSON API looks like for people to build a backend?

Reply:
Sunds good